### DISCUSSION TITLE
Just downloaded github

### DISCUSSION THREAD

**Discussion by mrgreen2015sa** [community/community#3762] [Mobile]
And I am trying to do the repository tutorial but the app isn't giving me the option to create one and follow along  .why is that someone help

**Comment by rafaelurben** [comment]
@mrgreen2015sa I think this is intentional design. Why would you want to create a repo on mobile?

**Comment by nplasterer** [comment]
Hi @mrgreen2015sa we don't currently support repository creation inside the apps. If you'd like to create a repository we'd suggest using the mobile web experience or pulling it up on a computer.
